sbncode/CAFMaker/CAFMakerParams.h

```diff
@@ -60,6 +60,12 @@ namespace caf
       Comment("List of suffixes to add to TPC reco tag names (e.g. cryo0 cryo1)")
     };
 
+    Atom<string> POTDataLabel {
```

Comment:
Can we call this `BNBPOTInfoLabel` or similar, to allow ourselves room for a `NuMIPOTInfoLabel` in future?